### ISSUE TITLE
Update to provide difference between PI and Manager

### ISSUE THREAD

**Issue by joachimweyl** [nerc-project/coldfront-plugin-cloud#112]
We need a way to differentiate between PIs and Managers that we can export from ColdFront.
Projects have multiple PIs. We need to know which PI is the Manager and which ones are not.
@Milstein please work with @knikolla to grow your ColdFront knowledge so that you can create this.

**Comment by knikolla** [comment]
@joachimweyl What do you mean by export from ColdFront?

**Comment by joachimweyl** [comment]
My understanding is that ColdFront is where this information is tracked. Does ColdFront have a way to track who is the PI and who has been designated as a Manager?

**Comment by knikolla** [comment]
Yes. There is exactly one PI per project, and there can be multiple managers. These are visible through the ColdFront dashboard. For invoicing purposes we expose the PI through the API when generating invoices.
